certdocs/0-DDLN-100-ORD.md

```diff
@@ -39,9 +39,9 @@ This document discusses the following topics:
 
 #### Internal Documents
```

Comment:
Not really related to this change specifically but thought I'd bring up. I know that ORD, SRD, SDD are 178 defined terms. However, I find it somewhat confusing to remember the map in my head between those three things and SysReqs, HLR, and LLRs. Would we consider renaming these MD files just SR, HLR, LLR, and then, if needed, always using the terms ORD/SRD/SDD in the generated reports only?

feel free to tell me that's crazy and I should just memorize the mapping, it just seems more clear.